tests/_support/AcceptanceTester.php

```diff
@@ -235,4 +235,19 @@ public function doAdministratorLogout()
 		$I->waitForText('Log in', 60, ['xpath' => "//fieldset[@class='loginform']//button"]);
 
 	}
+
+	/**
+	 * Selects an option in a Joomla Radio Field based on its label
+	 *
+	 * @return void
+	 */
+	public function selectOptionInRadioField($label, $option)
+	{
+		$I = $this;
+		$I->comment("Trying to select the $option from the $label");
+		$label = $webDriver->findField(['xpath' => "//label[contains(normalize-space(string(.)), '" . $label . "')]"]);
```

Comment:
findField() method is not working. I was also try to use webDriver Module but it's not work.